tools/windows/install-help/cal/FinalizeInstall.cpp

```diff
@@ -64,6 +64,19 @@ bool updateYamlConfig(CustomActionData &customActionData)
     return true;
 }
 
+bool writeInstallInfo(const CustomActionData &customActionData)
+{
+    std::wstring customInstallMethod = L"windows_msi_gui";
+    customActionData.value(L"OVERRIDE_INSTALLATION_METHOD", customInstallMethod);
+    WcaLog(LOGMSG_STANDARD, "Install method: %S", customInstallMethod.c_str());
+    std::wofstream installInfoOutputStream(installInfoFile);
```

Comment:
This will override the existing install_info file, right? We don't want that since it will erase the Chocolatey install info files.